iml-manager-cli/src/server.rs

```diff
@@ -299,13 +299,13 @@ async fn wait_till_agent_starts(
 }
 
 fn list_server(hosts: Vec<Host>, display_type: DisplayType) {
-    let term = Term::stdout();
-
     tracing::debug!("Hosts: {:?}", hosts);
 
-    let x = hosts.into_display_type(display_type);
-
-    term.write_line(&x).unwrap();
+    if !hosts.is_empty() || (display_type != DisplayType::Tabular) {
```

Comment:
This happens for all tables, can we generalize this to the trait output?

Reply:
I think yes.